frontend/src/feature/cards/components/CardDetailsComponent.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-card
+    class="pa-2 ma-2 mx-auto d-flex flex-column justify-space-between" max-width="500px" min-height="300px">
+    <v-card-title class="text-center">
+      {{ card.title }}
+    </v-card-title>
+    <v-card-subtitle>
+      <ul class="tags">
+        <li v-for="(tag, index) in card.tags">
+          <template v-if="index < card.tags!.length-1 ">
+            {{ tag }},
+          </template>
+          <template v-else>
+            {{ tag }}
+          </template>
+        </li>
+      </ul>
+    </v-card-subtitle>
+    <v-card-text>
+      <p>
+        {{ card.question }}
+      </p>
+      <p>
+        Answer: {{ card.correctOption }}
+      </p>
+    </v-card-text>
```

Comment:
Question should have label question in front,
then (the missing) options and correct option

=> Provide to each card detail data you display **also the field**, describing which data these are